bigbluebutton-html5/imports/api/users/server/handlers/validateAuthToken.js

```diff
@@ -45,7 +45,8 @@ export default function handleValidateAuthToken({ payload }) {
         addWelcomeChatMessage(meetingId, userId);
       }
 
-      return Logger.info(`Validated auth token as '${validStatus}' user=${userId} meeting=${meetingId}`);
+      return Logger.info(`Validated auth token as '${validStatus}'
+        user=${userId} meeting=${meetingId}`);
```

Comment:
Is there any difference in log/development.log? Any \n? That's the log that matters more. The console here is just for during development